appium/tests/specs/mock/inbox/ReadTextEmail.spec.ts

```diff
@@ -32,14 +32,19 @@ describe('INBOX: ', () => {
     mockApi.ekmConfig = {
       returnKeys: [ekmKeySamples.e2eValidKey.prv]
     }
+    mockApi.googleConfig = {
+      accounts: {
+        'e2e.enterprise.test@flowcrypt.com': {
+          messages: [GoogleMockMessage.CcAndBccTest],
```

Comment:
Could you use Ioan's suggestion here too? from https://github.com/FlowCrypt/flowcrypt-ios/pull/1700/files#r899791651

So it would be `messages: GmailApiMockMessageSubjectType`

Then instead of 

```ts
export enum GoogleMockMessage {
  CcAndBccTest = 'CC and BCC test'
}
```

it would be 

```ts
export type GoogleMockMessage = 'CC and BCC test';
// later:
export type GoogleMockMessage = 'CC and BCC test' | 'Some other test' | 'Yet another one' | ...
```

It makes the values exactly visible which is a good thing for test readability.

Reply:
Yep, will use it here too